src/include/processor/operator/copy/copy_node.h

```diff
@@ -4,6 +4,8 @@
 #include "storage/in_mem_storage_structure/in_mem_column.h"
 #include "storage/store/node_table.h"
 
+using namespace kuzu::storage;
```

Comment:
I think it is better to use `kuzu::storage::` rather than `using namespace` in the header file.